crates/cxx-qt-lib/src/core/qdatetime.rs

```diff
@@ -298,6 +301,16 @@ impl QDateTime {
         ffi::qdatetime_from_secs_since_epoch(secs, time_zone)
     }
 
+    /// Returns the datetime represented in the string as a QDateTime using the format given, or None if this is not possible.
+    pub fn from_string(string: &ffi::QString, format: ffi::DateFormat) -> Option<Self> {
```

Comment:
Always wonder if we should be calling it `string` or `qstring` in our API of cxx-qt-lib :thinking:

Reply:
Let's go with `from_string`, reference: QDate already has `from_string` methods :)